### PR TITLE
enable hotspot onboarding without location assert

### LINKED ISSUE #305: Missing skip location assert during initial hotspot setup
The new 3.0 app does not have the ability to skip location assert during initial setup. Hotspots can't be added to account now without a location set. Ver 2 was able to skip location for the hotspot to be added to the account.

### PR DESCRIPTION
**Pull request by tyler-whitman**
closes #305 

Enables a way to skip location assert when onboarding a hotspot. The new flow is shown below

Screen Shot 2021-03-01 at 6 35 36 PM

### CODE REVIEW

File: src/features/hotspots/setup/HotspotSetupLocationInfoScreen.tsx [hunk 1]
```diff
@@ -22,6 +23,10 @@ const HotspotSetupLocationInfoScreen = () => {
     }
   }
 
+  const skipLocationAssert = async () => {
```
Comment (matthewcarlreetz) [inline src/features/hotspots/setup/HotspotSetupLocationInfoScreen.tsx:26]: probably doesn't matter, but could wrap this in a `useCallback`

File: src/locales/en.ts [hunk 2]
```diff
@@ -396,6 +396,11 @@ export default {
       alertMessage:
         'Request to servers have timed out and we cannot add your Hotspot at this time.\n\nPlease contact support@helium.com and note MAC address %{mac}.',
     },
+    skip_location: {
```
Comment (matthewcarlreetz) [inline src/locales/en.ts:399]: We should add these to the other language files as well. cc: @cokes518

**Comment by matthewcarlreetz** [comment]
Looks good. My ble changes have caused some conflicts for you. Sorry :(

File: src/locales/en.ts (hunk 2 above)
Comment (cokes518) [reply src/locales/en.ts:399]: they'll auto populate when we send the en.json out for translation.

File: src/features/hotspots/setup/HotspotSetupLocationInfoScreen.tsx (hunk 1 above)
Comment (tyler-whitman) [reply src/features/hotspots/setup/HotspotSetupLocationInfoScreen.tsx:26]: Do we need it for things that dont use any state or anything? Would rather keep it simple if it doesn't really affect anything

**Review by matthewcarlreetz — APPROVED** [review]

**Pull request merged**